app/views/comments/create.js.erb

```diff
@@ -0,0 +1,11 @@
+let added_comment = $("<%= escape_javascript(render 'comments/comment', comment: @comment) %>");
```

Comment:
This line may be the one causing the issue. JP, it may be helpful to look at what the console prints out when you try creating the second comment.